dotcom-rendering/src/web/server/newslettersToHtml.tsx

```diff
@@ -0,0 +1,28 @@
+import createCache from '@emotion/cache';
+import { CacheProvider } from '@emotion/react';
+import { renderToString } from 'react-dom/server';
+import { extractNAV } from '../../model/extract-nav';
+import type { DCRNewslettersPageType } from '../../types/newslettersPage';
+import { NewslettersList } from '../components/NewslettersList';
+import { StandAlonePage } from '../layouts/StandAlonePage';
+import { populatePageTemplate } from './populatePageTemplate';
+
+export const newslettersToHtml = (model: DCRNewslettersPageType): string => {
+	const NAV = extractNAV(model.nav);
+	const key = 'dcr';
+	const cache = createCache({ key });
+	const html = renderToString(
```

Comment:
Rather than using this manually, you can now use `renderToStringWithEmotion` from lib/emotion.tsx which will put together this cache for you :)